Library/Formula/py2cairo.rb

```diff
@@ -27,16 +24,19 @@ class Py2cairo < Formula
   def install
     ENV.refurbish_args
 
+    # disable waf's python extension mode because it explicitly links
+    # libpython
+    inreplace "src/wscript", "pyext", ""
+    ENV["LINKFLAGS"] = "-undefined dynamic_lookup"
+    ENV.append_to_cflags `python-config --includes`
```

Comment:
:+1: Can we try and submit something (even if just an issue) upstream about this? Thanks!